lighthouse-core/test/test-utils.js

```diff
@@ -283,6 +285,14 @@ function isNode12SmallIcu() {
     Intl.NumberFormat.supportedLocalesOf('es').length === 0;
 }
 
+/**
+ * @param {ImportMeta} importMeta
+ * @return {string}
+ */
+function esmGetDirname(importMeta) {
```

Comment:
bikesheddable name :)

only added to this test-utils.js module b/c only using it in a test in this PR...but non-test files can use this too, so: ideas on where to keep? We're adding `root.js` ... maybe keep ESM utilities there?

Reply:
> only added to this test-utils.js module b/c only using it in a test in this PR...but non-test files can use this too, so: ideas on where to keep? We're adding `root.js` ... maybe keep ESM utilities there?

`dirname()` and keep it in the same file as `LH_ROOT` :P

Reply:
(actually don't need this yet)